deisctl/backend/fleet/list_units.go

```diff
@@ -72,6 +72,28 @@ var (
 	}
 )
 
+// unitPrefixes are compared to unit names, so "deisctl list" doesn't show "deis-demo.service",
+// for example. Update this list when adding a new Deis unit file.
+var unitPrefixes = []string{
```

Comment:
I did look at using a single data structure for exactly that purpose, but the Go packages that would consume it don't have any common imports. I'll take another look at it tomorrow--it would be lame to maintain essentially the same data in both places.